opossum-tool/OpossumUI
# Open locator popup with shortcut

### Summary of changes

With this PR the previously introduced popup to locate signals can be openend via the shortcut Ctrl+L or Cmd+L or via the Top bar. I also added a new icon for this functionality.

### Context and reason for change

We want to imlement a new feature to be able to locate signals based on criticality and certain licenses. 

### How can the changes be tested
Run the unittests.
Check out the PR and start OpossumUI. If you don't open a file the shortcut Ctrl+L or Cmd+L and will do nothing. But you will see the option "Locate Signals" in the top bar menu as in the screenshot below.
![image](https://github.com/opossum-tool/OpossumUI/assets/50019307/22176d6e-4c8f-4193-ae5e-7f09c05b061c)

Next, open a file and use Ctrl+L or Cmd+L and observe that the popup shows up as in the screenshot below. 
![image](https://github.com/opossum-tool/OpossumUI/assets/50019307/4f736cc8-78df-4bf8-b17b-e470fbcbb018)

Solves #1947.
